src/applications/static-pages/covidFaqLink.js

```diff
@@ -0,0 +1,23 @@
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+
+const insertLink = () => {
+  const linkParagraph = document.createElement('p');
+  linkParagraph.innerHTML =
+    'You can also use our <a href="/coronavirus-chatbot/">VA coronavirus chatbot</a> to get answers to your questions.';
+  const introText = document.querySelector('.va-introtext');
+
+  // Insert the link after the intro text
+  introText.parentNode.insertBefore(linkParagraph, introText.nextSibling);
+};
+
+export default function addLinkToCovidFAQ(store) {
+  // Only add the link once
+  let linkAdded = false;
+
+  store.subscribe(() => {
+    if (!linkAdded && toggleValues(store.getState()).covid19FaqChatbotLink) {
+      insertLink();
+      linkAdded = true;
+    }
+  });
```

Comment:
React was overkill for this. I figured we already had access to the Redux store, so...let's just use that, right? So I learned how to do that without `react-redux`. :grin: 

**Short story:** The initial state is `{ loading: true }`. It's not until after the network request for the feature toggles comes back that the state actually changes. So I had to create a listener and [subscribe](https://redux.js.org/api/store/#subscribelistener) to any updates to the redux store. This also meant I had to make sure we only added the link _once_.

Reply:
A couple suggestions - 

- `store.subscribe()` returns an `unsubscribe` function so you could use that instead of `linkAdded` as a flag.
- Why not use the same feature flag as https://github.com/department-of-veterans-affairs/vets-website/pull/12287?

Reply:
Ooh, good call with the `unsubscribe`! Didn't see that. :+1: 

We want to use different feature toggles for the two sites so we can throttle the traffic differently. The FAQ page (probably) has much less traffic than the facility locator page, so we'll likely want a much higher rate on it.

Reply:
Hrm. :confused: Well, I tried using `unsubscribe`, but couldn't get it to work like I wanted. Basically, I need `unsubscribe` in the context of the callback so it listens until the link is added, and then stops, but that's....not possible?

It was worth a shot, though. :man_shrugging:

Reply:
Here's an example - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/createMyVALoginWidget.js